nservicebus/upgrades/externalize-encryption.md

```diff
@@ -36,4 +35,23 @@ snippet: SplitDefiningEncryptedPropertiesAs
 
 ## Custom encryption Service
 
-snippet: SplitEncryptionFromIEncryptionService
\ No newline at end of file
+snippet: SplitEncryptionFromIEncryptionService
+
+
+## Compatibility
+
+The NServiceBus.Encryption.MessageProperty package is partially compatible with endpoints using NServiceBus package's encryption functionality:
+
+
+### Sending and receiving using NServiceBus.Encryption.MessageProperty
```

Comment:
Also I wonder if it'd be simpler to either only say this "NServiceBus.Encryption.MessageProperty **cannot** decrypt and encrypt messages with message properties of type `NServiceBus.Encryption.MessageProperty.WireEncryptedString`" or have sections "Supported" & "not supported". It's quite easy to miss the bolded **cannot** word right now.